### ISSUE TITLE
EVA modes of inheritance and ClinVar submitters

### ISSUE THREAD

**Issue by eric-czech** [opentargets/platform#1128]
We would like to be able to extract inheritance patterns (autosomal dominant, X-linked recessive, etc) specific to mendelian disorders as well as identify more heavily curated sources of associations to them within ClinVar like OMIM and ClinGen (i.e. we'd like to identify those submitters).  We also know how to get that kind of information directly from ClinVar, but I wanted to see if you guys have ever discussed the value of dimensions like this as part of a target ranking scheme.  

From our perspective, it would be ideal if this was incorporated as a part of the evidence strings provided via [eva-cttv-pipeline](https://github.com/EBIvariation/eva-cttv-pipeline).  Have data elements like this ever been requested before?

Thanks!

**Comment by d0choa** [comment]
It's on our immediate plans to increase the amount of information on our mendelian evidence. Mode of inheritance, mutation consequence, variant-disease confidence, variant IDs (when available) or increase coverage are some of the aspects we want to cover for different data sources. Of course, this will lead into improved scores for some of these evidence.

We'll work with EVA on improving eva-cttv-pipeline, but we will also need to work with some of the evidence sources such as gene2phenotype, OMIM or Genomics England PanelAPP.

It would help if you could share with us any wish list of feature you would like to see

**Comment by eric-czech** [comment]
Thanks @d0choa, I think that covers everything on our wishlist -- mode of inheritance, consequence (e.g. clinical significance), and variant-disease confidence (based on the source/submitter of the association).

When you say OMIM, do you mean anything other than the OMIM data collected as part of EVA from ClinVar?

**Comment by d0choa** [comment]
Yes, sorry I was not very specific. A subset of the OMIM data does not have variant information, therefore it does not come to us through EVA. However, this information is still relevant to establish target - disease relationships.

You will see a set of tickets for different bits of the work starting from Monday.

**Comment by eric-czech** [comment]
Hey @d0choa that's great to know -- what is that extra OMIM information?  Is it simply established gene - disease associations without knowledge of underlying variants or something less proximate?

I'll keep an eye out for those issues.  From my perspective, feel free to close this out if those new issues address this all in more detail.

**Comment by d0choa** [comment]
Yes that's the case for a subset of the OMIM evidence. I'm sure we will learn a bit more about them as we review them.
